torch/csrc/jit/runtime/static/impl.h

```diff
@@ -249,7 +248,7 @@ class TORCH_API StaticRuntime {
     return *outputs_[i];
   }
 
-  const std::vector<IValue*> outputs() const {
+  std::vector<IValue*> outputs() const {
```

Comment:
// Clang-Tidy: Return type 'const std::vector<IValue *>' is 'const'-qualified at the top level,
  // which may reduce code readability without improving const correctness